packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1000,6 +1000,52 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 		hint: 'See https://docs.astro.build/en/guides/content-collections/ for more on the `slug` field.',
 	},
 
+	/**
+	 * @docs
+	 * @message A collection queried via `getCollection()` does not exist.
+	 * @description
+	 * When querying a collection, ensure a collection directory with the requested name exists under `src/content/`.
+	 */
+	CollectionDoesNotExistError: {
+		title: 'Collection does not exist',
+		code: 9004,
+		message: (collection: string) => {
+			return `The collection **${collection}** does not exist. Ensure a collection directory with this name exists.`;
+		},
+		hint: 'See https://docs.astro.build/en/guides/content-collections/ for more on creating collections.',
+	},
+	/**
+	 * @docs
+	 * @message `COLLECTION_NAME` contains a mix of content and data entries. All entries must be of the same type.
+	 * @see
+	 * - [Defining content collections](https://docs.astro.build/en/guides/content-collections/#defining-collections)
+	 * @description
+	 * A content collection cannot contain a mix of content and data entries. You must store entries in separate collections by type.
+	 */
+	MixedContentDataCollectionError: {
+		title: 'Content and data cannot be in same collection.',
+		code: 9005,
+		message: (collection: string) => {
+			return `**${collection}** contains a mix of content and data entries. All entries must be of the same type.`;
+		},
+		hint: 'Store data entries in a new collection separate from your content collection, and share a `schema` in your content config.',
```

Comment:
```suggestion
		hint: 'Store data entries in a new collection separate from your content collection.',
```
As we discussed in the data collections PR, I hesitate to mention "sharing a schema" without an explicit example of what that means. Right now, the docs there show writing the schema and importing it in a separate file. I wouldn't be opposed to expanding that existing example to show TWO collections, one content and one data, using the same imported schema! (With a note that explicitly mentions "sharing a schema" when the structures are the same, even if the content type is different.)

So, I'd either truncate this (personally), OR add sharing a schema to the collections docs.